_common/core_ref.md

```diff
@@ -1,5 +1,21 @@
 ---
-layout: externalurl
-redirect_url: https://core.blockstack.org/
+layout: core
 title: "Stacks Node API"
+description: Interacting with the Blockstack Blockchain
+permalink: /:collection/:path.html
 ---
+
+With the launch of Stacks 2.0, a new version of the Blockstack blockchain was released. You currently have two ways of interacting with the blockchain:
+
+- Using a hosted HTTPS REST client that makes RPC calls to the blockchain
+- Directly making JSON RPC calls to the blockchain yourself
+
+Using the hosted client requires you to trust the client but provides a faster onboarding.
+
+## Hosted REST Client
+This client communicates with the JSON RPC endpoints. The hosted client references and OpenAPI specification can be find [here](https://blockstack.github.io/stacks-blockchain-sidecar/).
```

Comment:
- found*
- What do you mean by hosted client references?

I think I'd refer to this section as API Documentation. Redoc, the api doc generator we're using, has some features to query the API directly would don't think it's valuable to highlight this.

Sidenote, good hyperlink hygiene ensures a link makes sense in isolation of its context. Consider making a larger portion of this text the link, something like: [You can find the OpenAPI specification and documentation here.](6125026d256d44da0aff966ef82189ca30c1e7b1)

Would also add a big API Documentation link at the top of this page.